src/definitions/behaviors/form.js

```diff
@@ -1486,6 +1544,46 @@ $.fn.form.settings = {
     }
   },
 
+  formatter: {
+    date: function(date) {
+      var date = new Date(date);
+
+      return Intl.DateTimeFormat('en-GB', {
+        year: "numeric",
```

Comment:
Not needed. en-GB returns d/m/y by default

```javscript
return Intl.DateTimeFormat('en-GB').format(date);
```
is sufficient

Reply:
True. I really need some sleep !